carbonserver/carbonserver.go

```diff
@@ -472,7 +472,12 @@ func (listener *CarbonserverListener) updateFileList(dir string) {
 		return
 	}
 
-	freeSpace := stat.Bavail * uint64(stat.Bsize)
+	var freeSpace uint64
+	if stat.Bavail >= 0 {
+		freeSpace = uint64(stat.Bavail) * uint64(stat.Bsize)
+	} else {
+		freeSpace = 0
```

Comment:
`var freeSpace uint64` means the value of `freeSpace` is `0`. You should remove the `else` block entirely as it's not needed.

Reply:
^5 you're right, thanks!